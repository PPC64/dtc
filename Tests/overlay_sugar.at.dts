/dts-v1/;
/plugin/;

&foo {
	foo,status = "okay";
};

&bar {
	bar,status = "okay";
};

/ {
	fragment@0 {
		target = <&qux>;
		__overlay__ {
			qux,status = "okay";
		};
	};
};

&baz {
	baz,status = "okay";
	foobar: foobar {
		foobar,status = "okay";
	};
};

&{/} {
	foobar {
		foobar,status = "okay";
	};
};

&{/soc} {
	quux: baz {
		baz,status = "okay";
	};
};
