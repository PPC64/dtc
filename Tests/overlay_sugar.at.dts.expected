/dts-v1/;

/  {

	fragment@0 {

		target = <0xdeadbeef>;
		__overlay__ {

			foo,status = "okay";
		};
	};
	fragment@1 {

		target = <0xdeadbeef>;
		__overlay__ {

			bar,status = "okay";
		};
	};
	fragment@2 {

		target = <0xdeadbeef>;
		__overlay__ {

			qux,status = "okay";
		};
	};
	fragment@3 {

		target = <0xdeadbeef>;
		__overlay__ {

			baz,status = "okay";
			foobar {

				foobar,status = "okay";
				phandle = <0x1>;
			};
		};
	};
	__symbols__ {

		foobar = "/fragment@3/__overlay__/foobar";
	};
	__fixups__ {

		foo = "/fragment@0:target:0";
		bar = "/fragment@1:target:0";
		qux = "/fragment@2:target:0";
		baz = "/fragment@3:target:0";
	};
};
